src/Client.php

```diff
@@ -17,14 +17,16 @@
  */
 class Client
 {
+    public const SDK_VERSION = "1.0.0";
```

Comment:
how to make this dinamically after each release?

Reply:
it's simpler to update manually when releasing a new version, just change this specific line.

Reply:
it is possible to use git hooks and bash/php scripts with regex so when you run the `git tag` command to release a new version, edit this file with the new version and create commit